docs/en_US/ch1/exchangis_deploy_en.md

```diff
@@ -13,26 +13,26 @@ Exchangis installation is mainly divided into the following four steps ：
 
 | Dependent components                                                         | Must be installed | Install through train |
 |------------------------------------------------------------------------------| ------ | --------------- |
-| MySQL (5.5+)                                                                 | yes | [How to install mysql](https://www.runoob.com/mysql/mysql-install.html) |
-| JDK (1.8.0_141)                                                              | yes | [How to install JDK](https://www.runoob.com/java/java-environment-setup.html) |
-| Hadoop(2.7.2，Other versions of Hadoop need to compile Linkis by themselves.) | yes | [Hadoop stand-alone deployment](https://hadoop.apache.org/releases.html) ；[Hadoop distributed deployment](https://hadoop.apache.org/releases.html) |
-| Hive(2.3.3，Other versions of Hive need to compile Linkis by themselves.)     | yes | [Hive quick installation](https://hive.apache.org/downloads.html) |
+| JDK (1.8.0_141)                                                  | yes | [How to install JDK](https://www.oracle.com/java/technologies/downloads/) |
+| MySQL (5.5+)                                                  | yes | [How to install mysql](https://mysql.net.cn/) |
+| Hadoop(3.3.4，Other versions of Hadoop need to compile Linkis by themselves.) | yes | [Hadoop deployment](https://www.apache.org/dyn/closer.cgi/hadoop/common/hadoop-3.3.4/hadoop-3.3.4.tar.gz) |
```

Comment:
hadoop3.3.4 is sure?

Reply:
Hadoop was upgraded to 3.3.4 as with linkis1.4.0.